schema/logs/sample/nginx_access-log.json

```diff
@@ -0,0 +1,43 @@
+{
+  "@timestamp": "2022-12-09T10:39:23.000Z",
+  "observerTime": "2022-12-09T10:39:38.896Z",
+  "body": "47.29.201.179 - - [01/Mar/2020:10:34:43 +0100] \"GET / HTTP/1.1\" 200 612 \"-\" \"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.119 Safari/537.36\"",
+  "trace_id":"102981ABCD2901",
+  "span_id":"abcdef1010",
+  "attributes": {
+    "data_stream": {
+      "dataset": "nginx.access",
+      "namespace": "production",
+      "type": "logs"
+    }
+  },
+  "event": {
```

Comment:
Is there any special importance to event attributes. How do we identify if an index is built up of nginx logs and what dashboard templates to use?

Reply:
Is there any special importance to event attributes - yes its a part of the OTEL trace spec